webdriver/tests/interaction/element_clear.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success, assert_dialog_handled
+from tests.support.inline import inline
+
+
+def clear(session, element):
```

Comment:
The `element` argument name is misleading because it is actually a primitive web element reference.

It would be more consistent to pass the `Element` complex Python object instead of the web element reference here.

Reply:
fixed